src/routes/safe/components/Apps/communicator.ts

```diff
@@ -66,7 +66,7 @@ class AppCommunicator {
           this.send(response, msg.data.id)
         }
       } catch (err) {
-        console.log({ err })
+        console.error({ err })
```

Comment:
I guess no need for curly braces then.
There's also a new class [CodedException](https://github.com/gnosis/safe-react/blob/development/src/logic/exceptions/CodedException.ts) that we can use to log and track errors.

Reply:
Updated, please check if the usage is correct